### PR TITLE
feat: Runner spec list

### PR DESCRIPTION
**Pull request by ImCesar**
Created the flat view of specs list

### Demo

https://user-images.githubusercontent.com/8095629/140782894-3e0e2e44-9418-468c-b92c-3c440c01414d.mov

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by elevatebart** [comment]
The only remark I have is we are missing the hover smoothing.

**Review by lmiller1990 — COMMENTED** [review]
Cool, looks good, just a few minor comments

File: packages/graphql/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "@packages/types": "0.0.0-development",
     "@types/dedent": "^0.7.0",
     "chai": "^4.2.0",
+    "cypress-real-events": "1.4.0",
```
[the remark this answers is not in the document]
Comment (ZachJW34) [reply packages/graphql/package.json:34]: Yes it was incorrectly added here.

**Comment by ZachJW34** [comment]
@elevatebart added the hover smoothing!

File: packages/app/src/specs/InlineSpecListRow.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  } else {
+    if (!target.parentElement.firstElementChild) return
+
+    target.parentElement.firstElementChild.focus({ preventScroll: true })
+    target.parentElement.firstElementChild.scrollIntoView(false)
+  }
+}
+</script>
+
+<style>
```
Comment (JessicaSachs) [inline packages/app/src/specs/InlineSpecListRow.vue:75]:
```suggestion
<style scoped lang="scss">
```

File: packages/app/src/specs/SpecFileItem.vue [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  fileName: string
+  extension: string
+  selected: boolean
+}>()
+</script>
+<style>
+.group:hover path {
+  transition: all ease-in-out 0.3s;
+}
+</style>
```
Comment (JessicaSachs) [inline packages/app/src/specs/SpecFileItem.vue:29]: Can't merge until we fix this guy. We'll pair on it.

Comment (elevatebart) [reply packages/app/src/specs/SpecFileItem.vue:29]: ... aaaah `children:transition`! Always mischievous, never caught.

**Pull request merged**